HW_3/task_2.py

```diff
@@ -0,0 +1,8 @@
+#2. Дан список повторяющихся элементов. Вернуть список с дублирующимися элементами. В результирующем
+# списке не должно быть дубликатов.
+list = [1, 2, 4, 2, 3, 1, 5, 6, 6, 7, 7, 8, 3, 2, 1]
```

Comment:
Так ваш код просто выводит элементы исходного списка без повторений. Если будет так, то вывод не изменится
```suggestion
list = [1, 2, 4, 2, 1, 5, 6, 6, 7, 7, 8, 3, 2, 1]
```
Задача заключалась в другом